static/css/components/home.less

```diff
@@ -21,7 +21,6 @@ div.chartHome a:hover,
       .chartShow {
         width: 145px;
         height: 60px;
-        padding: 0 5px;
```

Comment:
I think we should use `margin: 0 5px;` instead of removing the `padding: 0 5px;`
Without padding:
![image](https://user-images.githubusercontent.com/64412143/116653577-1c094680-a9a5-11eb-8baa-8dbf202fc88e.png)
Using margin:
![image](https://user-images.githubusercontent.com/64412143/116653490-f11ef280-a9a4-11eb-8e7a-2c9aa709700d.png)

Reply:
Good point!